dds_cli/__main__.py

```diff
@@ -96,11 +96,23 @@
 )
 
 if len(sys.argv) == 1 or (len(sys.argv) > 1 and sys.argv[1] != "motd"):
-    motds = dds_cli.motd_manager.MotdManager.list_all_active_motds(table=False)
-    if motds:
-        dds_cli.utils.stderr_console.print("[bold]Important information:[/bold]")
-        for motd in motds:
-            dds_cli.utils.stderr_console.print(f"{motd['Created']} - {motd['Message']} \n")
+    try:
+        motds = dds_cli.motd_manager.MotdManager.list_all_active_motds(table=False)
+        if motds:
+            dds_cli.utils.stderr_console.print("[bold]Important information:[/bold]")
+            for motd in motds:
+                dds_cli.utils.stderr_console.print(f"{motd['Created']} - {motd['Message']} \n")
+    except dds_cli.exceptions.NoMOTDsError as no_motds_err:
+        LOG.info(no_motds_err)
```

Comment:
I would add some short comment here as well
```suggestion
        # Print message about no MOTD
        LOG.info(no_motds_err)
```